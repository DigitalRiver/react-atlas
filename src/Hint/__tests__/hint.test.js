import React from "react";
import { Hint } from "../index";
import renderer from "react-test-renderer";

describe("Test Hint component render", () => {
  it("Render correctly", () => {
    const tree = renderer.create(<Hint text="Some text" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
