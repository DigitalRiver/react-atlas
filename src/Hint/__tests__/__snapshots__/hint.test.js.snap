// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Hint component render Render correctly 1`] = `
<span
  className="hintLabel"
  style={undefined}
>
  Some text
</span>
`;
